Classes/NBNImageCaptureCell.m

```diff
@@ -1,68 +1,102 @@
 #import "NBNImageCaptureCell.h"
-
-static UIImagePickerController *imagePickerController;
+#import <AVFoundation/AVFoundation.h>
 
 @interface NBNImageCaptureCell ()
 @property (nonatomic) UIImageView *maskImageView;
-@property (nonatomic) CGSize cellSize;
+@property (nonatomic, weak) AVCaptureVideoPreviewLayer *previewCaptureLayer;
 @end
 
 @implementation NBNImageCaptureCell
 
-- (id)initWithFrame:(CGRect)frame
+#pragma mark - Init & dealloc
```

Comment:
I mean it's your code
but then please remove `#pragma mark - Setup SubViews` ;-)